bin/docker-build-proxy

```diff
@@ -14,6 +14,6 @@ rootdir="$( cd $bindir/.. && pwd )"
 . $bindir/_tag.sh
 
 # Default to a pinned commit SHA of the proxy.
-PROXY_VERSION="${PROXY_VERSION:-977ff25}"
+PROXY_VERSION="${PROXY_VERSION:- 51db6f8}"
```

Comment:
Thanks for bumping this! Looks like you've include an extra space character that shouldn't be there.

Reply:
Agh, thanks @klingerf!